arklex/types/resource_types.py

```diff
@@ -53,15 +55,23 @@ class ToolItem(Item):
     SHOPIFY_RETURN_PRODUCTS = "shopify/return-products"
 
     # HubSpot Tools
-    HUBSPOT_CHECK_AVAILABLE = "hubspot/check-available"
+    HUBSPOT_BOOK_MEETING = "hubspot/book-meeting"
```

Comment:
should we do ids like - `hubspot/meeting/book`, `hubspot/meeting/check-available`... I think this is a better way to organize ids as hubspot can have many tools